.github/workflows/publish-deb-brew-pkg.yml

```diff
@@ -28,3 +27,13 @@ jobs:
         tag: ${{ github.ref }}
     - name: Upload debian pkg to apt repository
       run: curl -F package=@target/debian/meilisearch.deb https://${{ secrets.GEMFURY_PUSH_TOKEN }}@push.fury.io/meilisearch/
+
+  homebrew:
+    name: Bump Homebrew formula
+    runs-on: ubuntu-latest
```

Comment:
Why do you specify ubuntu-latest here? Don't we want the binary to be compiled on Mac?
Or will Homebrew compile it for the different osx versions and that is only the platform on which we test the binary?

Reply:
The ubuntu is not used to compile the mac version. It's used to apply this command of brew:
``` bash
$ brew bump-formula-pr --strict meilisearch --url='https://github.com/meilisearch/MeiliSearch/archive/v0.9.0.tar.gz'
The same one I used manually to create a PR on HomeBrew repository. This kind of PR:
https://github.com/Homebrew/homebrew-core/pull/51906/files

The compilation from source code is done on HomeBrew's side.
```

Reply:
Nice, thank you for the details!